src/main/java/Duke.java

```diff
@@ -1,10 +1,86 @@
+import java.util.Scanner;
 public class Duke {
+    private static int taskCount = 0;
+    private static Task[] tasks = new Task[100];
+    private static final String dividingLine = "\n————————————————————————————————————————————————\n";
+    public static void printList(){
+        System.out.println(dividingLine + "Here are the tasks in your list:");
+        for(int i = 0; i < taskCount; i++){
+            System.out.println((i + 1) +". " + tasks[i].toString());
+        }
+        System.out.println("Now you have " + taskCount + " tasks in the list." + dividingLine);
+    }
+    public static void addTask(Task task){
+        tasks[taskCount] = task;
+    }
     public static void main(String[] args) {
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke" + System.lineSeparator() + "What can I do for you?" + dividingLine);
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while(!line.equals("bye")){
+            if(line.equals("list")){
+                printList();
+            }
+            else if(line.contains("mark") && !line.contains("unmark")){
```

Comment:
Why can't you just check for unmark first and then check for the mark, that way you don't have to specify the mark and unmark

Reply:
I do not think you need to check for "umarked" if you have already checked for "marked". However, the efforts are appreciated:)